File: Lab01/Python_Lang/Demo13.py
#
#  Created by NDHuy at 12/04/2020
#  Copyright 2020 by SEstudio
#  Check prime number
#

def main():
    n = int(input("Enter a integer number n > 0: "))

    if (n < 0):
        print("%d is not greater than Zero" %(n))
    elif (n < 2):
        print("%d is NOT a prime number" %(n))
    else:
        i = 2
        while (i <= (n / 2)):
            if (n % i == 0):
                break
            i += 1

        if (i >= (n / 2)):
            print("%d is a prime number" %(n))
        else:
            print("%d is NOT a prime number" %(n))


if __name__ == "__main__":
    main()

File: Session_01_BasicPagramming/Lab01/C_Lang/src/Demo13.cpp
//
//  Created by NDHuy at 12/04/2020
//  Copyright 2020 by SEstudio
//  Check prime number
//

#include <stdio.h>

int main()
{
    int n; 
    printf("Enter a integer n > 0: ");
    scanf("%d", &n);

    if (n <= 0)
        printf("%d is not greater than Zero", n);
    else
    {
        if (n < 2)
            printf("%d is NOT a prime number\n", n);
        else
        {
            int i;
            for(i = 2; i <= n / 2; i++)
            {
                if (n % i == 0)
                    break;
            }

            if (i >= (n / 2))
                printf("%d is a prime number\n", n);
            else
                printf("%d is NOT a prime number\n", n);
        }
    }

    return 0;
}